### PR TITLE
Update HOW-TO-GET-HELP.md : remove old syllabus links and

### PR DESCRIPTION
**Pull request by SallyMcGrath**
Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/code-style-guide/)

## Changelist

- removed links to old syllabus
- added links to new curriculum
- removed SDC 2022 channel list, which is very out of date

## Questions

@illicitonion Can you review this please? I am trying to set up a collection of open, merged, and closed PRs for learners to explore in the workshop described in the readme of this repo.

Please: 
- review this PR
- contribute one or more of your own
- please also edit and improve the workshop
- let's have some interesting activity for learners to explore on this repo

### CODE REVIEW

**Comment by SallyMcGrath** [comment]
👋 Hi explorers

What does this mean?

Screenshot 2024-12-15 at 18 46 03

File: HOW-TO-GET-HELP.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Update Instructions: https://gist.github.com/ChrisOwen101/84d82f03a60ce19f7f154a
 
 Learning to ask good questions is a key skill for a developer and you should practice this every week at CYF. You must spend a lot of time learning how to describe problems clearly and systematically:
 
-https://syllabus.codeyourfuture.io/guides/asking-questions
+https://curriculum.codeyourfuture.io/guides/asking-questions
```
Comment (illicitonion) [inline HOW-TO-GET-HELP.md:12]: There's one more link under the "Guide" title which points at the old syllabus - we should update that too (unfortunately GitHub doesn't let me comment on lines that weren't changed!)

**Review by illicitonion — APPROVED** [review]
Looks good to me, glad we're getting rid of the out of date links! Thanks!

I'll approve to unblock you, but I suggested one extra change it'd be good to do before merging :)

File: HOW-TO-GET-HELP.md (hunk 1 above)
Comment (SallyMcGrath) [reply HOW-TO-GET-HELP.md:12]: Thanks for this. Great catch. The guide it links to was removed some time ago. I have replaced it with some text that briefly recaps its contents.

I updated this PR by checking out this branch on my local machine and pushing changes to it.

**Comment by SallyMcGrath** [comment]
Screenshot 2024-12-16 at 10 37 25

I will now delete my merged branch.

**Pull request merged**